commerce_coordinator/apps/ecommerce/views.py

```diff
@@ -57,3 +60,45 @@ def get(self, request):
         )
 
         return Response(format_signal_results(results))
+
+
+class OrderCreateView(APIView):
+    """Accept Ecommerce request for creating a basket/order for a user."""
+    authentication_classes = (JwtAuthentication,)
+    permission_classes = (permissions.IsAuthenticated,)
```

Comment:
Since this endpoint (temporarily) allows a user to create orders on behalf of another user, can we mark it as an admin user endpoint only?

```suggestion
    permission_classes = (permissions.IsAdminUser,)  # TODO: Change to permissions.IsAuthenticated for UserOrderCreateView()
```

Reply:
Yup, Good suggestion.